automated_packaging/common_functions.pm

```diff
@@ -151,12 +151,13 @@ sub create_release_changelog {
                 if ($line =~ /^DESCRIPTION: */) {
                     $description_part = substr($line, length($&), -1);
                     if (length($description_part) > 78) {
-                        print("You have to shorten PR message $description_part of $pr_url");
-                        `git reset --hard`;
-                        die "Can not add description longer than 78 charachters";
+                        print("You have to shorten PR message $description_part of $pr_url\n");
+                        print("Can not add description longer than 78 charachters, please manually add this description\n");
```

Comment:
I suggest you add the warnings to the CHANGELOG as well.  It should be clear which entry is longer than the allowed 78 chars.

Reply:
hmm, you suggest me to add those ones to changelog but with a warning, makes sense

Reply:
then we manually remove those warnings and shorten those descriptions

Reply:
Please see https://github.com/citusdata/tools/pull/74#discussion_r422870340